Solomon Ucko  February 25, 2023
Â· Edited

  0  Collapse this comment
Copy link
In Python, you could do something like this to limit where you catch exceptions from:

try:
    name = await m_user.GetNameAsync()
except ...:
    m_label.Text = name
else:
    m_label.Text = "unknown"

The else block will run if the try block completes without throwing an exception, but any exceptions thrown by the else block are not caught.





pnoop star  February 24, 2023


  0  Collapse this comment
Copy link
Exceptions considered harmful